patches/flashrom-eace095b15eb034e42d97202cad70ce979d8ca38/0001-Add_RaptorPoint_PCH_support.patch

```diff
@@ -0,0 +1,237 @@
+From 99d16ef516b6080ceaaa866984b7e7fb471eaef0 Mon Sep 17 00:00:00 2001
```

Comment:
@i-c-o-n : under flashprog? I think yes.

Reply:
Different patch, but yes. It should work. Untested, though.